code/persian/index.ts

```diff
@@ -0,0 +1,45 @@
+const consonant = {
```

Comment:
Add this to the beginning of each file.

```suggestion
import { build, transform } from '~/base'

const consonant = {
```